k8s/perftest/common/probate/probate-caveats-fe.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-caveats-fe
+  namespace: probate
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
+spec:
+  releaseName: probate-caveats-fe
+  rollback:
+    enable: true
+    retry: true
+  chart:
```

Comment:
would be better to move to github chart sources really but not a blocker, you could do all in one go after

https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/rpe/draft-store-service/draft-store-service.yaml#L7